SeoPresentation.php

```diff
@@ -136,14 +136,9 @@ public function addExtractor(ExtractorInterface $extractor, $priority = 0)
     }
 
     /**
-     * Gets the SeoMetadata based on the content that contains the content.
-     *
-     * @param object $content
-     *
-     * @throws Exception\InvalidArgumentException
-     * @return SeoMetadata
+     * {@inheritDoc}
      */
-    private function getSeoMetadata($content)
+    public function getSeoMetadata($content)
```

Comment:
actually not when the method was private afaik. not in php, that is. you can't have another private method in an extending class with the same name, nor a protected/public one, as its declared private here. even if i am wrong, i think this change is minor enough that i can live with it - SeoPresentation needs not be extended, there are the extractors for customizign.

afaik Java handles that with less WTF on language level (but i guess a lot of WTF when actually running into that situation).